less/components/teach-like-mozilla-page.less

```diff
@@ -0,0 +1,32 @@
+.teach-like-mozilla {
+  h1, h2 {
+    margin: 0;
+    padding: 0;
+  }
+  .hero-unit h2 {
```

Comment:
these two rules should both be nested under `.hero-unit` instead of being written separately. (helps us quickly scan for complexity in later iterations)